main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Clooney")
+    Actor.objects.create(first_name="Keanu", last_name="Reeves")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    genre = Genre.objects.get(name="Dramma")
+    genre.name = "Drama"
+    genre.save()
+
+    actor = Actor.objects.get(first_name="George", last_name="Clooney")
+    actor.last_name = "Clooney"
+    actor.save()
+
+    actor = Actor.objects.get(first_name="Keanu", last_name="Reeves")
+    actor.first_name = "Keanu"
+    actor.last_name = "Reeves"
+    actor.save()
```

Comment:
Would make sense to break down main function is smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting..